lib/std/os/linux.zig

```diff
@@ -815,7 +809,12 @@ pub fn fchown(fd: i32, owner: uid_t, group: gid_t) usize {
 }
 
 pub fn fchmodat(fd: i32, path: [*:0]const u8, mode: mode_t, flags: u32) usize {
-    return syscall4(.fchmodat, @as(usize, @bitCast(@as(isize, fd))), @intFromPtr(path), mode, flags);
+    _ = flags;
+    return syscall3(.fchmodat, @bitCast(@as(isize, fd)), @intFromPtr(path), mode);
+}
+
+pub fn fchmodat2(fd: i32, path: [*:0]const u8, mode: mode_t, flags: u32) usize {
```

Comment:
Might be worth adding a comment about this fchmod vs the other one?  (And maybe note which Linux versions they came in on?)